policy/modules/services/xserver.te

```diff
@@ -32,6 +32,22 @@ gen_require(`
 ## </desc>
 gen_tunable(xdm_sysadm_login, false)
 
+## <desc>
+## <p>
+## Allows the X server to use TCP/IP
+## networking functionality (insecure).
+## </p>
+## </desc>
+gen_tunable(xserver_can_network, false)
+
+## <desc>
+## <p>
+## Allows the X display manager to use
+## TCP/IP networking functionality (insecure).
+## </p>
+## </desc>
+gen_tunable(xdm_can_network, false)
```

Comment:
Pleasee rename to `xserver_xdm_can_network`.

Reply:
Renamed